### PR TITLE
Integrated the following chains to safle-vault opBNB, Ronin, Sei Skale - Nebula, Immutable zk-EVM

### PR DESCRIPTION
**Pull request by Husienvora**
-Adding new default EVM chains

### CODE REVIEW

**Review by sshubhamagg — CHANGES_REQUESTED** [review]
1. Change PR heading
2. add names of integrated chains to changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -676,3 +676,7 @@
 -Resolved issue for unarchival of a wallet
 -Adding test for bitcoin
 -Adding sensitive info in env github pipeline
+
+### 2.9.3 (2024-12-10)
+
+-Adding new chains
```
Comment (apoorvq) [inline CHANGELOG.md:682]: Please put the name of the chains added

File: src/chains/index.js [hunk 2]
```diff
@@ -16,6 +16,11 @@ const evmChains = {
   zkEVM: { symbol: "ZKEVM", txType: 2 },
   bevm: { symbol: "BTC", txType: 0 },
   rootstock: { symbol: "RBTC", txType: 0 },
+  opbnb: { symbol: "BNB", txType: 0 },
+  sei: { symbol: "SEI", txType: 0 },
+  ronin: { symbol: "RON", txType: 0 },
+  nebula: { symbol: "sFUEL", txType: 0 },
+  immutable: { symbol: "IMX", txType: 0 },
```
Comment (apoorvq) [inline src/chains/index.js:23]: Is this Immutable or Immutable ZK?

File: .env
Comment (apoorvq) [inline .env:1]: This file is not supposed to be changed in this PR, please raise a separate PR for this. @Husienvora

**Review by apoorvq — CHANGES_REQUESTED** [review]
Left comments for some basic changes.

**Review by sshubhamagg — APPROVED** [review]

**Pull request merged**